tests/integrationv2/common.py

```diff
@@ -98,10 +107,12 @@ class Certificates(object):
     RSA_4096_SHA256 = Cert("RSA_4096_SHA256", "rsa_4096_sha256_client")
     RSA_4096_SHA384 = Cert("RSA_4096_SHA384", "rsa_4096_sha384_client")
     RSA_4096_SHA512 = Cert("RSA_4096_SHA512", "rsa_4096_sha512_client")
-    ECDSA_256 = Cert("ECDSA_256", "ecdsa_p256_pkcs1")
+
+    ECDSA_256 = Cert("ECDSA_256", "localhost_ecdsa_p256")
```

Comment:
What does the prefix `localhost` signify here? The naming convention used here is different from the current convention, maybe we could update the rest to follow suite? Ex: `ecdsa_p256_pkcs1` uses the older naming.

Reply:
The original certificate had a CN of example.com, and s2nc sends 'localhost' (or whatever we are trying to connect to) as the server name. This prevented us from verifying the server certificate in these situations. So I created new certificates that contain localhost as the CN (similar to the wildcard cert). 

I would suggest handling proper naming conventions in a new issue, and we can address all the certificates in this directory. #2119

Reply:
Thanks for creating the issue.